flow2ml.egg-info/SOURCES.txt

```diff
@@ -1,5 +1,6 @@
 README.md
 setup.py
+flow2ml/Auto_Results.py
```

Comment:
Please remove this. All this content gets generated when we publish package to Pypi

Reply:
I build the project on my local pc to test things out. So the sources.txt got updated automatically.